.changes/next-release/feature-AWSSDKforJavav2-130871b.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should this change entry be in this commit, since it refers to the Netty version?

Reply:
Yea, this change entry is extra. I didn't add the change entry in last pr so I added it here. Sorry about this :|